docs/create-specialized-question-types.md

```diff
@@ -0,0 +1,62 @@
+# Create Specialized Question Types
+
+Built-in question types are versatile and multi-functional, but in some cases, you need a question type with a more specific functionality. For example, you need to populate a [Dropdown](/Documentation/Library?id=questiondropdownmodel) question in advance. You can create a regular Dropdown question and specify its [`choices`](/Documentation/Library?id=questiondropdownmodel#choices) or [`choicesByUrl`](/Documentation/Library?id=questiondropdownmodel#choicesByUrl) property (depending on whether the choices come from a server or not). The following code shows a Country question configured in this manner:
```

Comment:
I first thought that the first approach is what you recommend (before I saw the "drawbacks" part). So maybe we should prepare the reader that we will show two ways to do this. 

```suggestion
Built-in question types are versatile and multi-functional, but in some cases, you need a question type with a more specific functionality. For example, you need to pre-populate a [Dropdown](/Documentation/Library?id=questiondropdownmodel) with a country list. You can use a regular Dropdown and customize it directly or you can create a specialized questions type. The first approach is more straightforward, but the second results in a more reusable solution. Let's consider both. 

The first example pre-populates a standard Dropdown question. You can specify the [`choices`](/Documentation/Library?id=questiondropdownmodel#choices) or [`choicesByUrl`](/Documentation/Library?id=questiondropdownmodel#choicesByUrl) property (depending on whether the choices come from a server or not). The following code shows a Country question configured in this manner:
```